modules/tinymce/src/plugins/accordion/main/ts/core/Actions.ts

```diff
@@ -0,0 +1,39 @@
+import { Id } from '@ephox/katamari';
+
+import Editor from 'tinymce/core/api/Editor';
+
+import * as Events from '../api/Events';
+
+const insertAccordion = (editor: Editor): void => {
+  const container = editor.selection.getNode();
+  if (container.nodeName === 'SUMMARY') {
+    const body = editor.dom.getNext(container, 'div.mce-accordion-body');
+    if (!body) {
+      return;
+    }
+    if (!body.lastChild) {
+      return;
+    }
+    editor.selection.setCursorLocation(body.lastChild, 1);
+  }
+
+  const uid = Id.generate('acc');
+  const summaryText = editor.dom.encode(editor.selection.getRng().toString() || editor.translate('Accordion summary...'));
+  const bodyText = editor.dom.encode(editor.translate('Accordion body...'));
```

Comment:
Wild idea, and this is a huge scope change, could we have the summary and body act like placeholder and be removed when the user focuses them? And also when saving?

We'd probably still need to translate the string, but it would avoid potentially having the text included on save.

This would be a new ticket, not an extension of the current one.

Reply:
We've had a discussion about the placeholders. The desired solution is to show them only if summary and body have no text within and hide otherwise. But we've decided to implement it as a separate task after implementing the accordion plugin.